protected/humhub/modules/user/controllers/ProfileController.php

```diff
@@ -129,7 +129,7 @@ public function actionFollowerList()
         $query->leftJoin('user_follow', 'user.id=user_follow.user_id AND object_model=:userClass AND user_follow.object_id=:userId', [':userClass' => User::class, ':userId' => $this->getUser()->id]);
         $query->orderBy(['user_follow.id' => SORT_DESC]);
         $query->andWhere(['IS NOT', 'user_follow.id', new Expression('NULL')]);
-        $query->active();
+        $query->visible();
```

Comment:
Move both of those queries to the `Follower` class. Controllers should not contain db query logic anyways.

Reply:
Commit https://github.com/humhub/humhub/pull/4879/commits/54b9428194fdc706860bdeceebf2773c6cbeeee0